sdk/objc/native/src/objc_video_encoder_factory.mm

```diff
@@ -15,6 +15,7 @@
 #import "base/RTCVideoEncoder.h"
 #import "base/RTCVideoEncoderFactory.h"
 #import "components/video_codec/RTCCodecSpecificInfoH264+Private.h"
+#import "components/video_codec/RTCCodecSpecificInfoH265+Private.h"
```

Comment:
In this file, all 265 related inclusion and call should be protected with OWT_USE_H265 macro .

Reply:
Done.